docs/user-manual/playcanvas-react/guide/xr.mdx

```diff
@@ -81,7 +94,7 @@ Here's a complete example with XR support, including buttons to enter AR/VR mode
   showDemo
   showControls={false}
 >
-  <Application>
+  <Application graphicsDeviceOptions={{ xrCompatible: true }}>
```

Comment:
Doesn't this already default to true when the engine initializes? I don't need this in Web Components.